<launch>
  <rosparam>
    names: [ "Wheel1", "Wheel2", "Wheel3" ]
    families: [ "Rosie" ]
  </rosparam>
  <node pkg="hebi_cpp_api_examples" type="omni_base_node" name="omni_base_node"/>
  <node pkg="hebi_cpp_api_examples" type="omni_base_controller_node" name="omni_base_controller_node" output="screen" required="true" />
</launch>
